modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -822,6 +822,49 @@ public Response setWebClip(@HeaderParam("Accept") String acceptHeader,
         }
     }
 
+	@POST
+	@Produces(MediaType.APPLICATION_JSON)
+	@Path("disenroll")
+	public Response setDisenrollment(@HeaderParam("Accept") String acceptHeader,
```

Comment:
While we allow any type of response via the accept header,  why do we also limit response to JSON?